app/BUILD.bazel

```diff
@@ -500,6 +520,47 @@ android_library(
     visibility = ["//visibility:public"],
 )
 
+kt_android_library(
+    name = "app",
+    custom_package = "org.oppia.app.ui",
+    srcs = APP_FILES,
+    enable_data_binding = 1,
+    resource_files = glob(LAYOUTS),
+    manifest = "src/main/AppAndroidManifest.xml",
+    deps = [
+        "//data",
+        "//model",
+        "//utility",
+        ":dagger",
+        ":databinding_adapters",
+        ":resources",
+        ":view_models",
+        ":views",
+        "databinding_resources",
+        artifact("androidx.appcompat:appcompat"),
+        artifact("com.github.bumptech.glide:glide"),
```

Comment:
Nit: prefer lexicographical ordering for deps.

Reply:
Done